pkg/sqlmodel/multirow.go

```diff
@@ -122,6 +122,152 @@ func GenDeleteSQL(changes ...*RowChange) (string, []interface{}) {
 	return buf.String(), args
 }
 
+// GenUpdateSQLFast generates the UPDATE SQL and its arguments.
+// Input `changes` should have same target table and same columns for WHERE
+// (typically same PK/NOT NULL UK), otherwise the behaviour is undefined.
+// It is a faster version compared with GenUpdateSQL.
+func GenUpdateSQLFast(changes ...*RowChange) (string, []interface{}) {
+	if len(changes) == 0 {
+		log.L().DPanic("row changes is empty")
```

Comment:
It seems we always use `log.Panic()` in TiCDC. Why do we need to get the current logger here? I'm not quite familiar with this API.

Reply:
It keeps the same `DPanic` usage as `GenUpdateSQL`, since this function is also used in DM. If we don't panic here(not in development mode), row changes will be lost.
@lance6716 Is it ok to change to `log.Panic()` here for DM?

Reply:
I think it's OK to use log.Panic.

@hi-rustin in DM when the log level is debug, we turn on Development flag.

https://github.com/pingcap/tiflow/blob/171e21b23af99b378bae4b394a1c0a9b80f98d5d/dm/pkg/log/log.go#L107-L118

and DM integration tests always use debug level log, so we can detect the misuse and control the blast radius when the bug is really happened at user's side.

Reply:
Changed to `log.Panic()`